auth_backend/modules/user/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class UserConfig(AppConfig):
-    name = 'modules.user'
+    name = 'apps.user'
```

Comment:
why was this changed to apps instead of modules

Reply:
it must be because of a merge. The apps folder was not renamed. Its fixed now.